ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2654,24 +2654,29 @@ patch_panel:
 
 ```yaml
 ptp:
-  mode: < mode >
+  mode: < boundary | transparent >
   forward_unicast: < true | false >
-  clock_identity: < clock-id >
+  clock_identity: < "clock-id in xx:xx:xx:xx:xx:xx format" >
   source:
-    ip: < source-ip>
-  priority1: < priority1 >
-  priority2: < priority2 >
-  ttl: < ttl >
-  domain: < integer >
+    ip: < source-ip >
+  priority1: < integer >
+  priority2: < integer >
+  ttl: < 1-254 >
+  domain: < 0-255 >
   message_type:
     general:
       dscp: < dscp-value >
     event:
-      dscp: < dscp-Value >
+      dscp: < dscp-value >
   monitor:
```

Comment:
please add
```
ptp:
  monitor:
    enabled: < true | false >
    sequence_id: < true | false >
```
Sequence_id is required (eos default off) to support the thresholds you are adding.

Reply:
Done